docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst

```diff
@@ -181,6 +181,43 @@ to optimize how Determined interacts with PBS:
    define ``CUDA_VISIBLE_DEVICES`` appropriately or provide the ``pbs.slots_per_node`` setting in
    your experiment configuration to indicate how many GPU slots are intended for Determined to use.
 
+-  Configure PBS to report GPU Accelerator type.
+
+   PBS administrators should set the value for ``resources_available.accel_type`` on each node with
+   a GPU. Otherwise, the Cluster tab on the Determined Web UI will show ``unconfigured`` for the
+   ``Accelerator`` field in the Resource Pool information.
+
+   PBS administrators can use the following set of commands to set the value of
+   ``resources_available.accel_type`` on a single node:
+
+   -  Check if the ``resources_available.accel_type`` value is set.
+
+   .. code:: bash
```

Comment:
It looks like the examples should be indented one more  unit to be inside the list.

Reply:
Indented as suggested. Updated image in the PR description.